frontend/plugins/insurance_ui/src/modules/insurance/components/products/ProductsColumns.tsx

```
@@ -70,14 +75,21 @@ export const createProductsColumns = (
     header: () => {
       const { t } = useTranslation('insurance');
-      return <RecordTable.InlineHead icon={IconShieldCheck} label={t('covered-risks')} />;
+      return (
+        <RecordTable.InlineHead
+          icon={IconShieldCheck}
+          label={t('covered-risks')}
+        />
+      );
     },
     cell: ({ cell }) => {
       const { t } = useTranslation('insurance');
```

ESLint

[error] React Hook "useTranslation" is called in function "cell" that is neither a React function component nor a custom React Hook function. React component names must start with an uppercase letter. React Hook names must start with the word "use".

(react-hooks/rules-of-hooks)